config/routes.rb

```diff
@@ -23,5 +23,11 @@
   put 'admins/:id/update_broker' => 'admins#update_broker', as: 'update_broker_admin'
   put 'admins/:id/update_buyer' => 'admins#update_buyer', as: 'update_buyer_admin'
 
+  # for pending broker signup accounts
+  get 'admins/pending/broker' => 'admins#pending_broker_signup', as: 'pending_broker_signup_admin'
+
+  # for approving pending broker signup accounts
+  put 'admins/:id/approve_broker' => 'admins#approve_broker_signup', as: 'approve_broker_admin'
+
```

Comment:
you may remove the unnecessary comments here since the responsibility of your routes are very obvious or easy to understand